### PR TITLE
Land Development Engineering Site Reviews Query

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/17718

New AMANDA query and [dataset](https://datahub.austintexas.gov/dataset/Land-Development-Engineering-Revisions/bek3-sdgd/about_data) I've created about LDE site plan reviews/revisions. 

This feeds a new tab in the license agreements Power BI report (testing workspace):
https://app.powerbigov.us/links/pjlZY4Rb1r?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare

#### Code
If you want to test the code, I have supplied a env_template, the [airflow DAG](https://github.com/cityofaustin/atd-airflow/blob/production/dags/dts_row_reporting.py) shows where each item exists in 1password.

Step 1. upload query as csv into S3
```
python amanda/amanda_to_s3.py --query lde_site_plan_revisions
```

Step 2. load data into socrata dataset
```
python metrics/s3_to_socrata.py --dataset lde_site_plan_revisions
```

#### Query
If you want to test the query by itself, you can copy `lde_site_plan_revisions` from queries.py into a DB viewer that supports Oracle. You can get the AMANDA read replica credentials in 1password as well.

#### Power BI
I've posted the link to the Power BI report above, feel free to poke around and see if things are making sense. Tracy also has requirements in attached issue.

#### Airflow
I'll make a companion PR for airflow after this is merged ✅

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 .env
 .DS_Store
 env_file
+/.idea
+*.pyc
```
Comment (johnclary) [inline .gitignore:6]: it looks like this repo needs a `.dockerignore` 👀

**Review by johnclary — APPROVED** [review]
cool—looks good to me. 

are these datasets public or are you a real deal Tyler Tech Enterprise Data Platform user?

File: metrics/socrata_config.py [hunk 2]
```diff
@@ -3,5 +3,10 @@
         "file_name": "license_agreements_timeline.csv",
         "resource_id": "arae-ym9d",
         "sodapy_method": "replace",
-    }
+    },
+    "lde_site_plan_revisions": {
+        "file_name": "lde_site_plan_revisions.csv",
+        "resource_id": "bek3-sdgd",
+        "sodapy_method": "upsert",
+    },
```
Comment (johnclary) [inline metrics/socrata_config.py:11]: one more thought: either here or in the `queries.py` file or perhaps the README it'd be nice to have a quick comment explaining what each of these measure/why they matter

**Comment by Charlie-Henry** [comment]
> are these datasets public or are you a real deal Tyler Tech Enterprise Data Platform user?

might as well make it public, idk what people would do with it though 🙈

**Pull request merged**